bin/run_multiple_panels.py

```diff
@@ -0,0 +1,600 @@
+import os
+import shutil             
+import sys
+import time
+import datetime as dt
+import subprocess
+from multiprocessing import Pool
+import pandas as pd
+
+from ochre import Dwelling, Analysis, ElectricVehicle
+
+# Script to run multiple simulations. Assumes each simulation has a unique folder with all required inputs
+
+# Download weather files from: https://data.nrel.gov/submissions/156 or https://energyplus.net/weather
+# weather_path = os.path.join('path', 'to', 'weather_files')
+weather_path = os.path.join(os.getcwd(), 'ResStockFiles', 'Weather')
+
+
+def run_multiple_hpc(main_folder, overwrite='False', n_max=None, *args):
+    # runs multiple OCHRE simulations on HPC using slurm
```

Comment:
@mnblonsky: Do you think it's worth keeping code for this? Maybe split out into a different file? It's useful for us internally to have this kind of code somewhere, but not sure we want to include it in an official release. At a minimum I'd probably suggest splitting this to a separate file?

Reply:
Looks like this is mostly taken from run_multiple.py. I think we shouldn't include it here. @JingWang-CUB if there's anything that you modified that you think should get updated in run_multiple.py, can you add it in there?

Reply:
I removed this function from this script.